radio/src/targets/flysky/CMakeLists.txt

```diff
@@ -36,11 +37,21 @@ if(PCB STREQUAL I6X)
   add_definitions(-DAFHDS2A_LQI_CH=14) # inject LQI at channel
 endif()
 
+if (CROSSFIRE)
+	add_definitions(-DCROSSFIRE)
+endif()
+
 if(PCBI6X_ELRSV2)
   add_definitions(-DPCBI6X_ELRSV2)
+  add_definitions(-DCROSSFIRE)
+  add_definitions(-DTELEMETRY_FRSKY)
 else() # features disabled for ELRSV2
-  add_definitions(-DPCBI6X_USB_MSD)
-  add_definitions(-DMENU_DIAGKEYS) # 364B
+#  add_definitions(-DPCBI6X_USB_MSD)
+#  add_definitions(-DMENU_DIAGKEYS) # 364B
```

Comment:
Why to remove it without a way to bring it back to any build? Can be optional but enable it by default.